### ISSUE TITLE
Site unresponsive upon redirecting to /install/database after completing web installer

### ISSUE THREAD

**Issue by SFFLAB** [freescout-helpdesk/freescout#322]
Title says it all, I've replicated this twice when installing on a fresh S3 instance. Once this happens, loading the domain, /login, /install, /install/database, etc. all lead to ERR_CONNECTION_REFUSED.

**Comment by freescout-helpdesk** [comment]
"The ERR_CONNECTION_REFUSED error is a client-side problem that can be caused by incorrect firewall, system or browser settings, but also by malware or a faulty Internet connection."

Double check the website URL you enter during installation (especially "https" part).

**Comment by SFFLAB** [comment]
I think I figured it out, it looks like installing CertBot for LE SSL certificates makes changes w/nginx (among other things) that ultimately causes HTTPS traffic to not work with FreeScout. I re-did this but instead stuck with HTTP for FreeScout, and put it behind an ELB for SSL, and that works fine.

I can't say if this is specific to me or how I installed it, or a general conflict between the two, however. But I have other instances running static pages and applications without issue that have CertBot/LE SSL implemented, and have never had an issue, so it's at least specific to how I went about installing them both.
